pyaedt/emit_core/results/revision.py

```diff
@@ -704,3 +704,22 @@ def set_emi_category_filter_enabled(self, category: EmiCategoryFilter, enabled:
             raise RuntimeError("This function is only supported in AEDT version 2024 R1 and later.")
         engine = self.emit_project._emit_api.get_engine()
         engine.set_emi_category_filter_enabled(category, enabled)
+
+    def license_session(self):
```

Comment:
I would suggest to either rename it `get_license_session` or use a property instead of a method.

Reply:
@MaxJPRey - Thanks for the idea! Changed it to a property and updated the test/docstring example accordingly.

Reply:
Great, thanks @bryankaylor .

Reply:
This is a pretty good explanation of properties vs. methods: https://stackoverflow.com/a/20506966

The checkout() called by this "property" definitely has side effects so it might be best to make license_session a method (`get_license_session()` as proposed). @bryankaylor - what do you think?

Reply:
Thanks @myoung301 , changed from a property to a get_license_session() function.